mobilecoind/src/subaddress_store.rs

```diff
@@ -45,7 +45,7 @@ impl SubaddressId {
     pub fn to_bytes(&self) -> [u8; 40] {
         let mut buf = [0u8; 40];
         buf[0..32].copy_from_slice(self.monitor_id.as_bytes());
-        buf[32..40].copy_from_slice(&self.index.to_le_bytes());
+        buf[32..40].copy_from_slice(&self.index.to_be_bytes());
```

Comment:
This is a breaking change and now feels like a good time to do it since the rest of the PR also is a breaking change. Using big endian instead of little endian means LMDB sorting works in a sane way, allowing to iterate over all SubaddressIds belonging to a given monitor id in the correct order, as well as allowing the iteration to start at a specific index.

Reply:
👍